terraform/gcp/variables.tf

```
@@ -15,7 +15,7 @@ variable "machine_type" {
 }
 
 variable "k8s_version" {
-  default = "1.16.9-gke.6"
+  default = "1.16.13-gke.1"
```

I think it's better to use according to the seacurity-related email we got from GCP https://github.com/w3f/infrastructure/issues/383 
```suggestion
  default = "1.16.13-gke.401"
```